lino/modlib/openui5/config/openui5/controller/App.controller.js

```diff
@@ -116,10 +133,25 @@ sap.ui.define([
             oDialog.open();
 
         },
+        /**
+         * Change the device type (Desktop or Mobile)
+         * @param oEvent
+         */
+        onChangeDeviceTypeButtonPress: function (oEvent) {
+            var oView = this.getView();
+            var oButton = oEvent.getSource();
+            var newdevice = 'phone';
+            if (this._selectedDevice === 'phone') {
```

Comment:
triple === for testing string equality? I would rather test using == instead of === (but I am not sure).

Reply:
It seems that  == operator test value equality and === operator tests equality in both type and value.
Examples:
console.log(1 == 1);
// expected output: true

console.log("1" == 1);
// expected output: true

console.log(1 === 1);
// expected output: true

console.log("1" === 1);
// expected output: false

More info 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators